recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -664,15 +727,6 @@ CONFIG_INOTIFYD=y
 CONFIG_LAST=y
 CONFIG_FEATURE_LAST_SMALL=y
 # CONFIG_FEATURE_LAST_FANCY is not set
-# CONFIG_LESS is not set
-CONFIG_FEATURE_LESS_MAXLINES=9999999
```

Comment:
You should leave all these CONFIG_FEATURE_LESS_* options as they were, or better, add them to the busybox_less USE flag.